src/main/java/core/basesyntax/FruitReportFile.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class FruitReportFile {
```

Comment:
not informative class name. This class violates `Single Responsibility` it reads file, creates list of fruits, and generates report